spec/backtrace_spec.rb

```diff
@@ -143,7 +160,38 @@
       end
 
       it "returns a properly formatted array of hashes" do
-        expect(described_class.parse(ex)).to eq(parsed_backtrace)
+        expect(
+          described_class.parse(ex, Logger.new('/dev/null'))
+        ).to eq(parsed_backtrace)
+      end
+    end
+
+    context "given an Oracle backtrace" do
+      let(:bt) do
+        [%(ORA-06512: at "STORE.LI_LICENSES_PACK", line 1945),
+         %(ORA-06512: at "ACTIVATION.LI_ACT_LICENSES_PACK", line 101),
+         %(ORA-06512: at line 2),
+         %(from stmt.c:243:in oci8lib_220.bundle)]
+      end
+
+      let(:ex) do
+        OCIError = AirbrakeTestError
+        OCIError.new.tap { |e| e.set_backtrace(bt) }
+      end
+
+      after { Object.__send__(:remove_const, :OCIError) }
```

Comment:
`OCIError = AirbrakeTestError` and this line look strange and probably deserve a comment.